hawk_test.go

```diff
@@ -120,32 +120,6 @@ func TestExpiration(t *testing.T) {
 	assert.Equal(t, http.StatusUnauthorized, recorder.Code)
 }
 
-func TestReplayProtection(t *testing.T) {
```

Comment:
interestingly this was actually still passing due to the request failing with an invalid hash error